src/components/PreviousEvents/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import SideEvent from '../SideEvent';
-import northernCapeCopy from '../../data/northernCape.json';
+import northernCapeCopy from '../../data/northernCapeHome.json';
```

Comment:
no don't double the number of files, rather add another field `description_short` to the existing files.